openlibrary/core/bookshelves.py

```diff
@@ -404,25 +509,35 @@ def get_sorted_reading_log_books(
                 oldb.query(query, vars=query_params)
             )
 
-            reading_log_work_keys = [
-                '/works/OL%sW' % i['work_id'] for i in reading_log_books
+            reading_log_keys = [
+                ('/works/OL%sW' % i['work_id'], '/books/OL%sM' % i['edition_id'])
+                if editions and i['edition_id']
+                else ('/works/OL%sW' % i['work_id'], None)
```

Comment:
If we don't actually care about the second element, we can create a single element tuple by adding `,` to the end. That prevents it from being treated as brackets/a string in this case.

```suggestion
                else ('/works/OL%sW' % i['work_id'],)
```

Reply:
This was a great tip to learn, extremely useful, thanks!  I ended up having to change this to a list in the final version, so it ended up unused, however.